src/main/java/io/nats/client/Connection.java

```diff
@@ -443,9 +443,46 @@
      * Gets a context for managing Jetstream streams
      * and consumers.
      * @param options JetStream options.
-     * @return a JetStream instance.
+     * @return a JetStreamManagement instance.
      * @throws IOException covers various communication issues with the NATS
      *         server such as timeout or interruption
      */
     JetStreamManagement jetStreamManagement(JetStreamOptions options) throws IOException;
+

```

Comment:
These are the interface methods added to the connection. They get a KV context and KV Management context. The division for management is that if it applies to the entire bucket, it is management.